Resources/Prototypes/Entities/Objects/Weapons/Guns/LMGs/lmgs.yml

```diff
@@ -69,7 +69,7 @@
   name: L6 SAW
   id: WeaponLightMachineGunL6
   parent: [BaseWeaponLightMachineGun, BaseSyndicateContraband]
-  description: A rather traditionally made LMG with a pleasantly lacquered wooden pistol grip. Uses .30 rifle ammo.
+  description: A rather traditionally-made Squad Automatic Weapon, or light machine gun, with a pleasantly-lacquered wooden pistol grip, though lacking a stock for controllability. Uses .30 rifle ammo.
```

Comment:
second added clause feels clunky and unnecessary imo

Reply:
Yeah, for sure. I was iffy on it; gonna reword it better